modules/search/client/components/SearchUsers.component.js

```diff
@@ -67,18 +67,16 @@ class SearchUsers extends React.Component {
     this.setState({ searchQuery: event.target.value });
   }
 
-  async actionSearch(event) {
+  actionSearch(event) {
     event.preventDefault();
     event.stopPropagation();
 
     this.setState({ isSearching: true }, () => {
-      searchUsers(this.state.searchQuery)
-        .then(({ data }) => {
-          this.setState({ users: data });
-        })
-        .finally(() => {
```

Comment:
@mrkvon do you know why `.finally` didn't work here?

@Akronix and I were expecting `searchUsers()` to return a standard promise and thus have `finally`. 🤔

Reply:
One possibility - you have a browser which [supports `Promise`](https://caniuse.com/#feat=promises), but [not `Promise.prototype.finally`](https://caniuse.com/#feat=promise-finally). That would also mean that we don't [transpile Promises](https://stackoverflow.com/questions/38960490/how-can-i-polyfill-promise-with-webpack) and we should.

Reply:
> Latest Firefox, should support ".finally" I suppose.

I can confirm the error before the fix.

Reply:
I think this is not critical. We can leave it as it is for now. My only concern is that the search members feature works and this isn't there yet. So I'd prioritize to have the indexing working.

Yep, the async was wrongly added to actionSearch, but we already removed it in this PR.